server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -1502,6 +1502,59 @@ public void setMessagesShouldSendAReadableHtmlMessage() throws Exception {
         calmlyAwait.atMost(30, TimeUnit.SECONDS).until( () -> isHtmlMessageReceived(recipientToken));
     }
 
+    @Test
+    public void setMessagesShouldComputeTextBodyWhenNoTextBodyButHtmlBody() throws Exception {
```

Comment:
No, it's GetMessageListMethodTest infact. We do not have GetMessagesTest and in the test, it called to "getMessageList" with the right fetch properties